app/services/WhatYouOweService.scala

```diff
@@ -43,18 +47,19 @@ class WhatYouOweService @Inject()(val financialDetailsService: FinancialDetailsS
     }
   }
 
-  def getWhatYouOweChargesList(isCodingOutEnabled: Boolean, isReviewAndReconcile: Boolean)
+  def getWhatYouOweChargesList(isCodingOutEnabled: Boolean, isReviewAndReconcile: Boolean, isFilterCodedOutPoas: Boolean)
                               (implicit headerCarrier: HeaderCarrier, mtdUser: MtdItUser[_]): Future[WhatYouOweChargesList] = {
     {
       for {
         unpaidChanges <- financialDetailsService.getAllUnpaidFinancialDetails(isCodingOutEnabled)
-      } yield getWhatYouOweChargesList(unpaidChanges, isCodingOutEnabled, isReviewAndReconcile)
+      } yield getWhatYouOweChargesList(unpaidChanges, isCodingOutEnabled, isReviewAndReconcile, isFilterCodedOutPoas)
     }.flatten
   }
 
   def getWhatYouOweChargesList(unpaidCharges: List[FinancialDetailsResponseModel],
                                isCodingOutEnabled: Boolean,
-                               isReviewAndReconciledEnabled: Boolean)
+                               isReviewAndReconciledEnabled: Boolean,
+                               isFilterCodedOutPoas: Boolean)
```

Comment:
not a big issue, but if we can use the same naming convention for isFilterCodedOutPoas?
ie. you can name it as isFilterCodedOutPoasEnabled or similar